canonical-web-and-design/react-components
# Fix storybook docs

## Done

- Reverts back storybook to 6.4.22 to fix broken docs tab.
- Drive-by: updates deprecated storybook imports

Fixes #790 

## QA

- Open demo https://react-components-791.demos.haus/
- Make sure storybook docs tab appears as expected

## Fixes

Fixes: #790 #792
